generator/views/metric_definitions_view.py

```diff
@@ -261,10 +269,7 @@ def to_lookml(self, v1_name: Optional[str], dryrun) -> Dict[str, Any]:
         # add the Looker dimensions
         for data_source, dimensions in base_view_dimensions.items():
             for dimension in dimensions:
-                if (
-                    dimension["name"] not in ignore_base_fields
-                    and dimension.get("type", "") != "time"
-                ):
+                if dimension["name"] not in ignore_base_fields:
                     dimension["sql"] = (
                         "${TABLE}." + f"{data_source}_{dimension['name']}"
                     )
```

Comment:
It seems like we could do all this logic in the same loop above, but a) probably a small performance difference for something not very time-sensitive so not a big deal, and b) not necessary to change in this PR anyway.